yukibot/src/Yukibot/Plugins/LinkInfo/Imgur.hs

```diff
@@ -69,6 +69,5 @@ fetchImgurTitle uri = do
   title <- liftHandler fetchTitle uri
   case title of
     Title t
-      | T.map toLower t == "imgur" -> return NoTitle
-      | T.map toLower t == "imgur: the simple image sharer" -> return NoTitle
+      | (T.pack "imgur:") `T.isInfixOf` (T.toLower t) -> return NoTitle
```

Comment:
`OverloadedStrings` is on, so the `T.pack` call isn't needed there as string literals are overloaded.